webapp/app/components/pages/UsersForm/UsersForm.jsx

```diff
@@ -1,31 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Row } from 'reactstrap';
 import FormInput from '../../common/form/FormInput';
-
+import { Row } from 'reactstrap';
+import { get } from 'lodash';
```

Comment:
Replace this with 
`import get from 'lodash.get';`

Reply:
I try to do the same with lodash.cloneDeep but I am having some issues: warning webpack-bundle-analyzer > bfj-node4@5.3.1: Switch to the `bfj` package for fixes and new
features!